Resources/Prototypes/Entities/Mobs/NPCs/silicon.yml

```diff
@@ -204,6 +204,70 @@
     interactFailureString: petting-failure-honkbot
     interactSuccessSound:
       path: /Audio/Items/bikehorn.ogg
+  - type: MovementSpeedModifier
+    baseWalkSpeed: 4
+    baseSprintSpeed: 6
+
+- type: entity
+  parent: MobSiliconBase
+  id: MobSyndieBot
+  name: syndiebot
+  description: Death to Nanotrasen!
+  components:
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Silicon/Bots/syndiebot.rsi
+    state: syndiebot
+  - type: GhostRole
+    makeSentient: true
+    name: ghost-role-information-syndiebot-name
+    description: ghost-role-information-syndiebot-description
+  - type: GhostTakeoverAvailable
+  - type: InnateTool
+    tools:
+      - id: WeaponAdvancedLaser
+      - id: EnergyDagger
+  - type: Hands
+    showInHands: false
+  - type: Body
+    prototype: Bots
+  - type: CombatMode
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 40
+      behaviors:
+      - !type:TriggerBehavior
+    - trigger:
+        !type:DamageTrigger
+        damage: 60
```

Comment:
60 damage to destroy it seems to less, and I doubt anybody would buy it for 10 TC

Reply:
with advanced laser now its a pretty solid option for 10 TC

Reply:
> 60 damage to destroy it seems to less, and I doubt anybody would buy it for 10 TC

During this time, he will kill several people)

Reply:
> but like it's super noticeable, you won't get to any head without being destroyed or getting a lot of damage, like idk

There is a stealth function for this.